media/admin/en-GB.plg_vmshipment_zasilkovna.ini

```diff
@@ -101,6 +106,11 @@ PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_INVALID_TYPE="There is field with invali
 PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_WEIGHT_PRICE_MISSING="Weight price in weight rules is missing"
 PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_WEIGHT_MISSING="Weight in weight rules is missing"
 PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_WEIGHT_EXCEEDED="Weight in weight rules is over specified limit"
+PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_NO_HD_CARRIER_SELECTED="Choose a home delivery carrier"
+PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_HD_CARRIER_IS_OUT_OF_ALLOWED_COUNTRIES="Country of this Home Delivery Carrier is not allowed allowed countries."
+PLG_VMSHIPMENT_PACKETERY_SHIPPING_ERROR_HD_CARRIER_IS_IN_BLOCKING_COUNTRIES="Country of this Home Delivery Carrier is among blocking countries."
```

Comment:
Home Delivery Carrier/Výdejní místa prosím všude malými písmeny

Reply:
beru to jako jmena terminu, ktery se u zasilkovny bezne pouzivaji